scaffolder-templates/github-workflows/convert-workflow-to-template/skeleton/templates/${{values.workflow_id}}/${{values.workflow_id}}.yaml

```diff
@@ -11,16 +11,26 @@ spec:
   system: red-hat-developer-hub
   type: service
 
-  # matches workflow's dataInputSchema
-  parameters: ${{values.parameters}}
+  parameters:
+    - title: Choose Target Entity
+      properties:
+        target_entity:
+          title: Target Entity for the workflow run
```

Comment:
IIUC, the entity picker filters only for components.
Shouldn't the title and description be changed to specify that?
meaning replace "Entity" with "Component".